rln/src/main.rs

```diff
@@ -6,33 +6,17 @@ use ark_std::rand::thread_rng;
 use color_eyre::Result;
 
 use ark_bn254::Bn254;
-use ark_groth16::{
-    create_random_proof, generate_random_parameters, prepare_verifying_key, verify_proof,
-};
-
 use num_bigint::BigInt;
 
 // Tracing
 use ark_relations::r1cs::{ConstraintLayer, ConstraintTrace, TracingMode};
-//use tracing::{event, span, Level};
 use tracing_subscriber::layer::SubscriberExt;
 
 // JSON
 use serde::Deserialize;
-//use serde_json;
-
-#[derive(Debug, Deserialize)]
-//#[serde(rename_all = "camelCase")]
-struct WitnessInput {
```

Comment:
Sure we don't need this anymore?

Reply:
I moved the data structure in protocol.rs https://github.com/vacp2p/zerokit/blob/6d3571034df74e786102c06cdf4e1b78a4015094/rln/src/protocol.rs#L31 since there can be populated with some helper functions with no need to expose all fields.